Dockerfile

```diff
@@ -1,27 +1,74 @@
-FROM kyero/ruby
+FROM ruby:2.6.3
```

Comment:
Could this perhaps be a descendent of our `docker-ruby` image?

Reply:
what does it change?